### ISSUE TITLE
Could I run netlify-cms-proxy-server as express middleware in gatsby?

### ISSUE THREAD

**Issue by laurenskling** [netlify/netlify-cms#3348]
I've just upgraded from `netlify-cms-backend-fs` to https://www.netlifycms.org/docs/beta-features/#working-with-a-local-git-repository

`netlify-cms-backend-fs` exposed something I could hook into my Gatsby Dev server:
```
const fsMiddleware = require('netlify-cms-backend-fs/dist/fs');
exports.onCreateDevServer = ({ app }) => {
  fsMiddleware(app);
};
```

This means I only have to start `gatsby develop` to make local development work with a local git backend.

The new feature `netlify-cms-proxy-server` needs to be run standalone according to the docs. Is it possible to run it within my gatsby dev server? I don't like having to run 2 different separated processes to make development work.

**Comment by laurenskling** [comment]
Just ran the fix presented in #3361 and it works beautifully. Thanks!

**Comment by erezrokah** [comment]
Thanks for the update!
